app/memory/memory_ui_service.py

```
@@ -259,9 +260,36 @@ async def update_memory(
+            reviewer_exists = False
+            try:
+                reviewer_lookup = await supabase._exec(
+                    lambda: supabase.client.schema("auth")
+                    .table("users")
+                    .select("id")
+                    .eq("id", reviewer_id_str)
+                    .maybe_single()
+                    .execute()
+                )
+                reviewer_exists = bool(reviewer_lookup and reviewer_lookup.data)
+            except Exception as exc:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)